docs/docs/default-actions.mdx

```diff
@@ -2,47 +2,60 @@
 id: default-actions
 sidebar_label: Default Actions
 title: Default Actions
+abstract: Default actions are actions that are built into the dialogue manager by default. Most of these are automatically predicted based on certain conversation situations. You may want to customise these to personalise your assitant.
 ---
 
-<a aria-hidden="true" tabIndex="-1" className="anchor enhancedAnchor" id="default-actions"></a>
+Each of these actions have a default behaviour, described in the sections below.
+In order to overwrite this default behavior, write a [custom action](./custom-actions.mdx)
+whose `name()` method returns the same name as the default action:
```

Comment:
Question: how does this work for non-sdk action servers? I guess that the name needs to make it into the registered list of action names from `/actions`?

Reply:
would say that's not something we need to cover here